#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'drc_task_common'

try:
    import imp
    imp.find_module(PACKAGE)
    from dynamic_reconfigure.parameter_generator_catkin import *;
except:
    import roslib; roslib.load_manifest(PACKAGE)
    from dynamic_reconfigure.parameter_generator import *;

from math import pi

gen = ParameterGenerator ()
gen.add("min_z", double_t, 0, "minimum z", 0.5, 0.0, 2.0)
gen.add("max_z", double_t, 0, "maximum z", 1.0, 0.0, 2.0)

exit (gen.generate (PACKAGE, "drc_task_common", "FilterBBoxPosition"))
